pubpub/pubpub
# update: Integrate Workflow editor and Submissions listing into dashboard

This PR makes the Submission Workflow editor and the Submissions listing coexist on the same dashboard tab. I've settled on using two different routes/container for this: the listing is at `/dash/collection/:slug/submissions` and the workflow editor is at `/dash/collection/:slug/submissions/worfklow`. When you click on the Submissions tab for the first time you'll be redirected to create a workflow. This has the nice side effect of using the breadcrumbs as they were intended:

<img width="757" alt="image" src="https://user-images.githubusercontent.com/2208769/148415504-31a4b3a4-14d4-4b6f-bebf-efc1a7444280.png">

While I was in here I also added a button on the Submissions listing to enable/disable submissions:

<img width="348" alt="image" src="https://user-images.githubusercontent.com/2208769/148415634-f2e2610e-fc72-428a-b185-84443611f3f6.png">

_Test plan:_
- For a Collection without a `SubmissionWorkflow`, clicking the Submissions tab should redirect you to the workflow creator. 
- Click "Save and Continue" to be redirected to the submissions listing
- Check that the "Edit Workflow" button takes you back to the workflow editor, which allows you to make changes
- Check that the enable/disable submissions button works.
